### DISCUSSION TITLE
how to train a model of siamrpn

### DISCUSSION THREAD

**Discussion by 1085052159** [STVIR/pysot#419] [General]
我只用VID单独训练RPN，训练了20个epoch后，在OTB2015上，AUC只有0.18左右，求大佬指教一下如何训练SiamRPN，小弟感激不尽

**Comment by ZhiyuanChen** [comment]
单目标跟踪网络是难以训练的，很遗憾我不能在这方面给予你帮助。

**Comment by 1085052159** [comment]
> 单目标跟踪网络是难以训练的，很遗憾我不能在这方面给予你帮助。

没事，感谢大佬的回复
